.dev/docker/env/Dockerfile

```diff
@@ -23,19 +23,18 @@ RUN apt-get update \
       clang \
       wget
 
-RUN if [ "$USE_CUDA" = "true" ] ; then wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | tar xz \
-     && apt install -y libblas-dev \
-     && cd eigen-3.3.7 \
-     && mkdir build \
-     && cd build \
-     && cmake .. \
-     && make install \
-     && cd ../.. \
-     && rm -rf eigen-3.3.7/ \
-     && rm -f eigen-3.3.7.tar.gz ; \
-    fi
+RUN wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | tar xz \
+ && apt install -y libblas-dev \
+ && cd eigen-3.3.7 \
+ && mkdir build \
+ && cd build \
+ && cmake .. \
+ && make install \
+ && cd ../.. \
+ && rm -rf eigen-3.3.7/ \
+ && rm -f eigen-3.3.7.tar.gz
 
-RUN if [ "$USE_CUDA" = "true" ] ; then apt install -y software-properties-common \
+RUN if python -c "exit(0 if `gcc -dumpversion | cut -c 1-3` < 7.0 else 1)"; then apt install -y software-properties-common \
```

Comment:
Why not use shell comparison along with `cut -d. -f1`?

Reply:
bash doesn't support floating-point math. I could just do integer comparison but used python just in case in the future we want to do floating point comparision.

Reply:
You don't need floating point op. If you supply `.` as delimited and get the first field, that's the major version